README.md

```diff
@@ -7,6 +7,8 @@
 
 A standalone service for Knative to pause/unpause containers when request count drops to zero.
 
+**NOTE** It is highly recommended to disable agressive probing on your application when using the container-freezer. This can be done by setting `periodSeconds: 0` on your application's readiness probe. See [here](https://github.com/psschwei/sleeptalker/blob/main/sleeptalker.yaml#L14-L15) for an example.
```

Comment:
```suggestion
**NOTE** It is highly recommended to disable aggressive probing on your application when using the container-freezer. This can be done by setting `periodSeconds: 0` on your application's readiness probe. See [here](https://github.com/psschwei/sleeptalker/blob/main/sleeptalker.yaml#L14-L15) for an example.
```